src/tigerbeetle/main.zig

```diff
@@ -162,11 +162,13 @@ const Command = struct {
         });
     }
 
-    pub fn start(arena: *std.heap.ArenaAllocator, args: *const cli.Command.Start) !void {
+    pub fn start(base_allocator: std.mem.Allocator, args: *const cli.Command.Start) !void {
+        var counting_allocator = vsr.CountingAllocator.init(base_allocator);
+
         var traced_allocator = if (constants.tracer_backend == .tracy)
-            tracer.TracyAllocator("tracy").init(arena.allocator())
+            tracer.TracyAllocator("tracy").init(counting_allocator.allocator())
         else
-            arena;
+            &counting_allocator;
```

Comment:
I think the types are different now? One is concreet allocator, the other is allocator vtable

Reply:
I mean, due to comptime it doesn't matter that the types are different, but this should break downstream code? Although tests pass, so I must be wrong. Could you double check that it still compiles with tracy?

Reply:
ah, no, just can't read as usual!